dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/sms/config/SmsGateway.java

```diff
@@ -110,36 +110,28 @@ public HttpStatus send( String urlTemplate, HttpEntity<?> request, HttpMethod ht
         {
             response = restTemplate.exchange( urlTemplate, httpMethod, request, klass );
 
-            if ( response != null )
-            {
-                statusCode = response.getStatusCode();
-            }
-            else
-            {
-                log.error( "Server response is null" );
-                statusCode = HttpStatus.INTERNAL_SERVER_ERROR;
-            }
+            statusCode = response.getStatusCode();
         }
         catch ( HttpClientErrorException ex )
         {
-            log.error( "Client error", ex );
+            log.error( "Sms request Client error", ex );
```

Comment:
Can you please lower case the former first word of the sentence. Same in some of the logs below.